R/GaussSuppressionFromData.R

```diff
@@ -512,19 +540,45 @@ GaussSuppressionFromData = function(data, dimVar = NULL, freqVar=NULL,
     }
     uniqueCharVar <- charVar[!(charVar %in% dVar)]
     if (length(uniqueCharVar)) {
+      #if (length(uniqueCharVar) == 1) {
+      #  charData <- aggregate(data[uniqueCharVar], data[unique(dVar)], function(x) x[1])
```

Comment:
Is this code part needed?

Reply:
Thanks , I will remove in final submission to CRAN